### PR TITLE
Added improved definition for MDCV that matches other industry group description

### PR DESCRIPTION
**Pull request by digitaltvguy**
1. Specific language about "should" include mDCV for PQ and SDR
2. What to do for SDR where the min/max display values are unknown (populate from Table 11 of ITU-T Series H Supp 19).  This matches description from other industry group.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3884,8 +3884,13 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
           the Mastering Display Color Volume (mDCv) used at the point of content creation, 
           as specified in [[SMPTE ST 2086]]. The mDCv provides informative static metadata to 
           allow a destination (consumer) display to optimize it's tone and color mappings based 
-          on its inherent capabilities. Additional examples of its most common use-cases are available 
-          in [[ITU-T Series H Supplement 19]].</p>
+          on its inherent capabilities. The MDCV chunk should be included for both PQ and SDR 
+          images. It is less common for HLG images. Color Primaries and White Point characteristics 
+          can be derived from CICP chunk formats. Specific examples of its most common use-cases
```
Comment (ProgramMax) [inline index.html:3889]: Can you rename `MDCV` to `mDCv` and `CICP` to `cICP` since they both refer to the chunk?

File: index.html [hunk 2]
```diff
@@ -3884,8 +3884,13 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
...
           as specified in [[SMPTE ST 2086]]. The mDCv provides informative static metadata to 
           allow a destination (consumer) display to optimize it's tone and color mappings based 
-          on its inherent capabilities. Additional examples of its most common use-cases are available 
-          in [[ITU-T Series H Supplement 19]].</p>
+          on its inherent capabilities. The MDCV chunk should be included for both PQ and SDR 
+          images. It is less common for HLG images. Color Primaries and White Point characteristics 
+          can be derived from CICP chunk formats. Specific examples of its most common use-cases 
+          for both HDR and SDR are available in [[ITU-T Series H Supplement 19]].</p>
+
+          <p> For SDR images, if MDCV display min/max luminance are unknown, the default
```
Comment (ProgramMax) [inline index.html:3892]: Here, too, should be `mDCv`.

File: index.html [hunk 3]
```diff
@@ -3884,8 +3884,13 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
...
           allow a destination (consumer) display to optimize it's tone and color mappings based 
-          on its inherent capabilities. Additional examples of its most common use-cases are available 
-          in [[ITU-T Series H Supplement 19]].</p>
+          on its inherent capabilities. The MDCV chunk should be included for both PQ and SDR 
+          images. It is less common for HLG images. Color Primaries and White Point characteristics 
+          can be derived from CICP chunk formats. Specific examples of its most common use-cases 
+          for both HDR and SDR are available in [[ITU-T Series H Supplement 19]].</p>
+
+          <p> For SDR images, if MDCV display min/max luminance are unknown, the default 
+          characteristics can be derived from the values in ITU_T Series H Supplemental 19 Table 11 .”</p>
```
Comment (ProgramMax) [inline index.html:3893]: Should ITU_T be ITU-T?

**Review by ProgramMax — APPROVED** [review]
Thank you.
I have some suggested changes.
But I'm approving since these can be changed later and the wording is useful as-is.

**Comment by palemieux** [comment]
@ProgramMax GitHub will not show me the diff. Do you have the same issue?

**Comment by ProgramMax** [comment]
I see the diff when I select "Files changed" up at the top.

But, @digitaltvguy you see how this pull request shows 18 commits from 2 months ago? That is because your copy of the repo at https://github.com/digitaltvguy/PNG-spec needs updated. Under the green "Code" button you should see a button that says "Sync fork". Don't click it yet. But once we land this pull request, click it.

What you'll want to do is sync your fork every time you're about to make a change. Then you're adding your change on top of the latest.

**Pull request merged**